examples/hits_example.py

```diff
@@ -0,0 +1,82 @@
+import os
+from typing import Any
+
+import graphblas as gb
+import graphblas_algorithms as ga
+
+import numpy as np
+import scipy.sparse as sps
+from numpy.testing import assert_allclose
+
+os.environ["SPARSE_BACKEND"] = "Finch"
+import sparse
+
+# select namespace
+xp = sparse  # np jnp
+Array = Any
+
+
+def converged(xprev: Array, x: Array, N: int, tol: float) -> bool:
+    err = xp.sum(xp.abs(x - xprev))
+    return err < xp.asarray(N * tol)
+
+
+class Graph:
+    def __init__(self, N: int, A: Array):
+        assert A.ndim == 2 and all(dim == N for dim in A.shape)
+        self.N = N
```

Comment:
Maybe it makes sense to infer `N` from `A` instead of an `assert` to make sure it's consistent? Or maybe we can directly pass an array in and use its `.shape` property?

Reply:
Right, we don't need that - done!